docs/dev-setup.md

```diff
@@ -0,0 +1,42 @@
+---
+id: dev-setup
+title: Setup Environment
+---
+
+Gotify requires:
+
+- Go 1.11.5
+- Node 11.x
+
+## Clone sources
+
+Clone gotify server source from git:
+
+```bash
+$ git clone https://github.com/gotify/server.git && cd server
+```
+
+## Setup Backend
+
+If you are in GOPATH, enable [go modules](https://github.com/golang/go/wiki/Modules) explicitly:
+
+```bash
+$ export GO111MODULE=on
+```
+
+Download dependencies
+
+```bash
+$ GO111MODULE=off make download-tools
```

Comment:
GO111MODULES would be already turned off in https://github.com/gotify/server/pull/125